dist/profile/manifests/jiraveg.pp

```diff
@@ -0,0 +1,32 @@
+# profile for jira
+class profile::jiraveg (
+  $jira_javahome,
+  $jira_version
+){
+  $pkgs = ['openjdk-7-jre-headless', 'facter']
+  package { $pkgs :
+    ensure    => present
+  } ->
+  class { 'jira':
+    version     => $jira_version,
+    javahome    => $jira_javahome,
+    installdir  => '/opt/jira',
+    homedir     => '/srv/jira_home',
+    user        => 'jira_user',
+    group       => 'jira',
+
+    # Database Settings
+    db          => 'postgresql',
```

Comment:
It was first step to expose settings, just to keep under eye what available and what we need configure.